WcaOnRails/app/javascript/image-preview/index.js

```
@@ -0,0 +1,15 @@
+import loadImage from 'blueimp-load-image';
+
+$(() => {
+  $('.image-preview-container').each((index, container) => {
+    const $container = $(container);
+    const $input = $($container.data('inputSelector'));
+    $input.on('change', () => {
+      loadImage(
+        $input[0].files[0],
+        img => img.type !== 'error' && $container.empty().append(img),
```

I'm not in love with silently failing to do stuff. How do you feel about adding a `if...else` that does a `console.error` if there's a problem?